optashift-employee-rostering-gwtui/src/main/resources/org/optaplanner/openshift/employeerostering/gwtui/client/resources/i18n/OptaShiftUIConstants_en.properties

```diff
@@ -1,6 +1,7 @@
 AbstractRosterViewPanel.startSolving=Click the <i>Solve</i> button to start solving.
 AbstractRosterViewPanel.finishedSolving=Solving finished
 AbstractRosterViewPanel.solve=Solve
+AbstractRosterViewPanel.terminateEarly=Terminate
```

Comment:
Would you mind calling this label "Terminate early"? That would be consist with all the other example UI's and the API.

Reply:
It was originally "Terminate early", but the button has a weird spacing issue where one space look like three and is aesthetically displeasing. I can change it back, it doesn't look as good.